versions/spack.ver

```diff
@@ -6,7 +6,7 @@ export jasper_ver=2.0.32
 export libpng_ver=1.6.37
 export zlib_ver=1.2.13
 export esmf_ver=8.5.0
-export fms_ver=2023.02.01
+export fms_ver=2023.04
```

Comment:
This will change the `fms_ver` for all platforms if changed in `spack.ver`. Is that the intent? If not, please add `export fms_ver=2023.04` to the noaacloud run version files.